docs/transpile/defaults_and_configuration_options.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Transpilation defaults and configuration options
+description: Default settings and configuration options including optimization and approximation degree
+
+---
+
+# Transpilation default settings and configuration options
+
+Transpilation is the process of rewriting a given input circuit to match the topology of a specific quantum device, to optimize the circuit for execution on present day noisy quantum systems, or both. You can pass circuits to the `transpile()` method and use all default settings, or you can fine-tune the transpilation by using the available parameters. 
+
+## All available parameters:
+
+Following are all of the available parameters for the `transpile()` method.  All parameters except `circuits` are optional.  For full details, see the (Transpiler API documentation)[https://docs.quantum-computing.ibm.com/api/qiskit/transpiler#transpiler-api].
+
+Do we want this whole list?  If so, I'll add descriptions from (https://docs.quantum-computing.ibm.com/api/qiskit/compiler#compilation-routines)
```

Comment:
Would it be worth grouping the arguments according to which stage of transpilation they correspond to?

Reply:
Oooh - I just put out a version like that :)